lib/salus/scanners/base.rb

```diff
@@ -332,6 +332,23 @@ def create_list_file_option(keyword:, value:, prefix:, separator:, suffix:, join
       "#{prefix}#{keyword}#{separator}#{Shellwords.escape(validated_files.join(join_by))}#{suffix}"
     end
 
+    def fetch_exception_ids
+      exceptions = @config.fetch('exceptions', [])
+      ids = []
+      exceptions.each do |exception|
+        except = Salus::ConfigException.new(exception)
+        unless except.valid?
+          report_error(
+            'malformed exception; expected a hash with keys advisory_id, changed_by, notes',
+            exception: exception
+          )
+          next
+        end
+        ids << except.id if except.active?
```

Comment:
It should be `except.advisory_id`, not `except.id`, right?
because in another file, you have
```ruby
validates :advisory_id
```